### PR TITLE
doma: introduce options for prebuilt graphics

### PR DESCRIPTION
**Pull request by iusyk**
ANDROID_PREBUILT_DDK=yes will trigger the system
to not fetch closed source graphics and use prebuilt binaries. Assuming that all prebuilt should be placed into eva directory in the root of the product.

### CODE REVIEW

File: prod-cockpit-rcar.yaml [hunk 1]
```diff
@@ -543,3 +546,22 @@ parameters:
...
+    "yes":
+      overrides:
+        variables:
+          XT_DOMA_DDK_KM_PREBUILT_MODULE: "../android/eva/eva/pvr-km/pvrsrvkm.ko"
+          XT_DOMA_SOURCE_GROUP: "default"
+        components:
+          doma:
+            builder:
+              env:
+                - "DDK_UM_PREBUILDS=../android/eva/eva/pvr-um"
```
Comment (arminn) [inline prod-cockpit-rcar.yaml:567]: I don't think path ../android/eva/eva/pvr-um is a correct path

File: prod-cockpit-rcar.yaml [hunk 2]
```diff
@@ -543,3 +546,22 @@ parameters:
...
+    "no":
+      default: true
+      overrides:
+        variables:
+          XT_DOMA_DDK_KM_PREBUILT_MODULE: "../android_kernel/out/android12-5.4/staging/vendor/lib/modules/pvrsrvkm.ko"
+          XT_DOMA_SOURCE_GROUP: "all"
+    "yes":
+      overrides:
+        variables:
+          XT_DOMA_DDK_KM_PREBUILT_MODULE: "../android/eva/eva/pvr-km/pvrsrvkm.ko"
```
Comment (arminn) [inline prod-cockpit-rcar.yaml:561]: I don't think path ../android/eva/eva/pvr-km/pvrsrvkm.ko is a correct path

**Comment by arminn** [comment]
What component creates ../android/eva/eva dir ????

**Comment by iusyk** [comment]
> What component creates ../android/eva/eva dir ????

1st , build.ninja has the definition of out_dir: android/eva
2nd archive has the following structure 
eva
  pvr-km
  pvr-um

The test
out_dir = android/eva
build.ninja has the line: command = mkdir -p $out_dir && tar -m -C $out_dir -xf $in

so, if we run the following
tar -m -C android/eva -xf ./rcar-prebuilts-graphics-xt-doma.tar.gz 
we have the following result:
 ls android/eva/
  eva

**Comment by arminn** [comment]
@iusyk you need to find out why there 2 eva dir's, this is not default behavior(since prod-devel doesn't have that eva/eva). Maybe some options like "dir: eva" is wrong.

**Comment by iusyk** [comment]
> @iusyk you need to find out why there 2 eva dir's, this is not default behavior(since prod-devel doesn't have that eva/eva). Maybe some options like "dir: eva" is wrong.

well, the reason :  file tar.gz has the folder eva, this is why we have double 'eva/eva' folders.

**Comment by arminn** [comment]
RB

**Pull request merged**